code/app-util-xdm/build.gradle

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+plugins {
+    id 'com.android.library'
+}
+
+android {
+    namespace 'com.adobe.marketing.tester.xdm.commerce'
+
+    compileSdk rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        minSdk rootProject.ext.minSdkVersion
+        targetSdk rootProject.ext.targetSdkVersion
+    }
+
+    compileOptions {
+        sourceCompatibility rootProject.ext.sourceCompatibility
+        targetCompatibility rootProject.ext.targetCompatibility
+    }
+}
+
+dependencies {
+    implementation project(':edge')
```

Comment:
If we move this xdm https://github.com/adobe/aepsdk-edge-android/blob/dev-v2.0.0/code/edge/src/main/java/com/adobe/marketing/mobile/xdm/Property.java
 to the tester package we can remove edge project dependency right?

Reply:
we cannot move it, is part of the Edge SDK

Reply:
Move as in create a copy of it in the tester package.

Reply:
Does not make much sense to copy them. The Edge extension provides them for Schema implementors, hence the dependency on Edge SDK for this util lib.